src/ftsh_functions.c

```diff
@@ -0,0 +1,112 @@
+/***************************************************************************//**
+
+	@file         ftsh_functions.c
+
+	@author       Max Bronckers
+
+	@date         Thursday, 1 May 2018
+
+	@brief        FTSH functions
+
+*******************************************************************************/
+
+#include "ftsh.h"
+#include "ftsh_functions.h"
+
+/*
+	List of builtin commands, followed by their corresponding functions.
+ */
+char *builtin_str[] = {
+	"help",
+	"exit",
+	"load"
+};
+
+int (*builtin_func[]) (char **) = {
+	&ftsh_help,
+	&ftsh_exit,
+	&ftsh_load
+};
+
+/**
```

Comment:
This header comment style conflicts with Christina's in the `match` header. Please standardize on one of these styles. I don't care which as long as it's consistent.